src/js/media/flash.js

```diff
@@ -153,6 +153,10 @@ vjs.Flash.prototype['currentSrc'] = function(){
   }
 };
 
+vjs.Flash.prototype['currentSource'] = function(){
```

Comment:
why not add it to the parent of these techs. That way, you can have it only in one place.